pkgs/development/python-modules/siuba/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, hypothesis
+, numpy
+, pandas
+, psycopg2
+, pymysql
+, python-dateutil
+, pytz
+, pyyaml
+, six
+, sqlalchemy
+}:
+
+buildPythonPackage rec {
+  pname = "siuba";
+  version = "0.4.2";
+  disabled = pythonOlder "3.7";
+
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "machow";
+    repo = "siuba";
+    rev = "v${version}";
+    hash = "sha256-Q2nkK51bmIO2OcBuWu+u7yB8UmaqiZJXpuxXcytTlUY=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    pandas
+    psycopg2
+    pymysql
+    python-dateutil
+    pytz
+    pyyaml
+    six
+    sqlalchemy
+  ];
+
+  checkInputs = [
+    hypothesis
+    pytestCheckHook
+  ];
+  doCheck = false;
+  # requires running mysql and postgres instances; see docker-compose.yml
```

Comment:
is it possible to disable just the db based tests?

Reply:
There's not an obvious way such as test names or file names indicating separate backends which could simply be skipped with a regex.  It could potentially be hacked somehow.

Reply:
That sounds reasonable (skipping the check entirely I mean), maybe consider adding a `pythonImportsCheck` just to ensure the module can be loaded.